integration_tests/integration/serviceProviderReferrals.cy.js

```diff
@@ -1420,7 +1418,7 @@ describe('Service provider referrals dashboard', () => {
             'Session details': 'Session 2',
             'Date and time': '10:02am on 31 Mar 2021',
             Status: 'scheduled',
-            Action: 'Reschedule sessionGive feedback',
+            Action: 'Reschedule session\n\t\t\t\t\t\t\n\t\t\t\t\t\t\t\tGive feedback',
```

Comment:
including all the control characters looks messy and could result in brittle tests if the layout is changed at all.
could we change these `.should('deep.equal'` tests to something like 
```js
        .should(result => {
          expect(result).to.have.length(2)
          expect(result[0]).to.deep.equal({
            'Session details': 'Session 1',
            'Date and time': '9:02am on 24 Mar 2021',
            Status: 'completed',
            Action: 'View feedback form',
          })
          expect(result[1]).to.deep.include({
            'Session details': 'Session 2',
            'Date and time': '10:02am on 31 Mar 2021',
            Status: 'scheduled',
          })
          expect(result[1].Action).to.match(/^Reschedule session[\n|\t]*Give feedback$/)
        })
```